stm32cube/stm32f3xx/README

```diff
@@ -36,14 +36,20 @@ License Link:
 
 Patch List:
 
-   *Changes from official delivery:
-    -dos2unix applied
-    -trailing white spaces removed
-
-    *Fix warnings for extraneous parentheses
-      Using clang 7.0.1, if ((htim->State == HAL_TIM_STATE_BUSY))
-      generates warnings.  Remove the extra parentheses
-     Impacted files:
-      drivers/src/stm32f3xx_hal_tim.c
-      drivers/src/stm32f3xx_hal_tim_ex.c
-     ST Bug tracker ID: 63616
+[master 6e66c60] stm32cube: update stm32f3 cube version
+ 178 files changed, 144238 insertions(+), 129522 deletions(-)
+ rename stm32cube/stm32f3xx/drivers/include/{stm32f3xx_hal_conf.h => stm32f3xx_hal_conf_template.h} (73%)
+ create mode 100644 stm32cube/stm32f3xx/drivers/include/stm32f3xx_hal_exti.h
+ rewrite stm32cube/stm32f3xx/drivers/include/stm32f3xx_hal_pcd.h (66%)
+ rewrite stm32cube/stm32f3xx/drivers/include/stm32f3xx_hal_pcd_ex.h (74%)
+ rewrite stm32cube/stm32f3xx/drivers/include/stm32f3xx_hal_smartcard_ex.h (77%)
+ rewrite stm32cube/stm32f3xx/drivers/include/stm32f3xx_hal_tim_ex.h (85%)
+ rewrite stm32cube/stm32f3xx/drivers/include/stm32f3xx_hal_usart_ex.h (66%)
+ create mode 100644 stm32cube/stm32f3xx/drivers/include/stm32f3xx_ll_usb.h
+ create mode 100644 stm32cube/stm32f3xx/drivers/src/stm32f3xx_hal_exti.c
+ rewrite stm32cube/stm32f3xx/drivers/src/stm32f3xx_hal_i2s_ex.c (78%)
+ rewrite stm32cube/stm32f3xx/drivers/src/stm32f3xx_hal_nand.c (64%)
+ rewrite stm32cube/stm32f3xx/drivers/src/stm32f3xx_hal_opamp_ex.c (75%)
+ rewrite stm32cube/stm32f3xx/drivers/src/stm32f3xx_hal_pcd_ex.c (72%)
+ create mode 100644 stm32cube/stm32f3xx/drivers/src/stm32f3xx_hal_usart_ex.c
+ create mode 100644 stm32cube/stm32f3xx/drivers/src/stm32f3xx_ll_usb.c
```

Comment:
I don't see a particular added value in this.
Maybe a link to official package delivery note could be given though

Reply:
Then, do we link to [https://github.com/STMicroelectronics/STM32CubeF3/blob/master/Release_Notes.html]
or copy the content in its 'Main Changes' section

Reply:
or do we include the  entire Release Note from _'STM32CubeF3/Release_Notes.html_' directly in the '_modules/hal/stm32/stm32cube/stm32f3xx_' folder as a '_release_note.html_' file